src/test/java/seedu/storemando/model/item/LocationContainsKeywordsPredicateTest.java

```diff
@@ -0,0 +1,79 @@
+package seedu.storemando.model.item;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.storemando.testutil.ItemBuilder;
+
+public class LocationContainsKeywordsPredicateTest {
+
+    @Test
+    public void equals() {
+        List<String> firstPredicateKeywordList = Collections.singletonList("first");
+        List<String> secondPredicateKeywordList = Arrays.asList("first", "second");
+
+        LocationContainsKeywordsPredicate firstPredicate =
+            new LocationContainsKeywordsPredicate(firstPredicateKeywordList);
+        LocationContainsKeywordsPredicate secondPredicate =
+            new LocationContainsKeywordsPredicate(secondPredicateKeywordList);
+
+        // same object -> returns true
+        assertTrue(firstPredicate.equals(firstPredicate));
+
+        // same values -> returns true
+        LocationContainsKeywordsPredicate firstPredicateCopy =
+            new LocationContainsKeywordsPredicate(firstPredicateKeywordList);
+        assertTrue(firstPredicate.equals(firstPredicateCopy));
+
+        // different types -> returns false
+        assertFalse(firstPredicate.equals(1));
+
+        // null -> returns false
+        assertFalse(firstPredicate.equals(null));
+
+        // different item -> returns false
+        assertFalse(firstPredicate.equals(secondPredicate));
+    }
+
+    @Test
+    public void test_locationContainsKeywords_returnsTrue() {
+        // One keyword
+        LocationContainsKeywordsPredicate predicate =
+            new LocationContainsKeywordsPredicate(Collections.singletonList("Apple"));
+        assertTrue(predicate.test(new ItemBuilder().withLocation("Apple Banana").build()));
```

Comment:
For this part just let Fazil know to change this to standardised locations, because this one he may overlook thinking that it's correct cause Apple Banana etc are what we using for item names.

Reply:
All right noted!